README.md

```diff
@@ -1 +1,73 @@
-# videojs-verify-generator-plugin
+# videojs-generator-verify
+
+[![Build Status](https://travis-ci.org/brightcove/videojs-generator-verify.svg?branch=master)](https://travis-ci.org/brightcove/videojs-generator-verify)
+[![Greenkeeper badge](https://badges.greenkeeper.io/brightcove/videojs-generator-verify.svg)](https://greenkeeper.io/)
+[![Slack Status](http://slack.videojs.com/badge.svg)](http://slack.videojs.com)
+
+[![NPM](https://nodei.co/npm/videojs-generator-verify.png?downloads=true&downloadRank=true)](https://nodei.co/npm/videojs-generator-verify/)
+
+A tool to verify that a videojs-generate-plugin project is ready for publish.
+
+Maintenance Status: Stable
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Installation](#installation)
+- [Inclusion](#inclusion)
+- [Basic Usage](#basic-usage)
+- [License](#license)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Installation
+
+Install `videojs-generator-verify` and `in-publish` via npm
+
+```sh
+$ npm install --save-dev videojs-generator-verify in-publish
+```
+
+Then add a script to your `package.json` as follows:
+
+```json
+{
+  "scripts": {
+    "prepublish": "not-in-install && npm run build && vjsverify || in-install"
+  }
+}
+```
+
+## Usage
+
+### Command line
+This package provides three binaries, most are just an alias for `videojs-generator-verify`:
+`vjsverify`, `videojs-verify`, and `videojs-generator-verify`
```

Comment:
Not sure it makes sense to expose more than one binary.

Reply:
yeah lets just make it `vjsverify`